docs/detections/api/exceptions/api-update-exception-item.asciidoc

```diff
@@ -1,5 +1,6 @@
 [[exceptions-api-update-item]]
 === Update exception item
+//Check if this endpoint can also be used to update rule default exceptions.
```

Comment:
@yara can [this endpoint](https://security-docs_bk_4021.docs-preview.app.elstc.co/guide/en/security/master/exceptions-api-update-item.html) be used to update rule exception items that were added to rule default lists or shared exception lists?